README.md

```diff
@@ -0,0 +1,88 @@
+# @hmcts/cmc-draft-store-middleware
+
+[![Travis badge](https://api.travis-ci.org/hmcts/cmc-draft-store-middleware.svg?branch=master)](https://travis-ci.org/hmcts/cmc-draft-store-middleware)
+
+[![Codecov badge](https://codecov.io/github/hmcts/cmc-draft-store-middleware/coverage.svg?branch=master)](https://codecov.io/github/hmcts/cmc-draft-store-middleware?branch=master)
+
+[![NPM version badge](https://img.shields.io/npm/v/@hmcts/cmc-draft-store-middleware.svg)](https://www.npmjs.com/@hmcts/cmc-draft-store-middleware)
+
+[![Node version badge](https://img.shields.io/node/v/@hmcts/cmc-draft-store-middleware.svg)](https://www.npmjs.com/@hmcts/cmc-draft-store-middleware)
+
+[![Greenkeeper badge](https://badges.greenkeeper.io/hmcts/cmc-draft-store-middleware.svg)](https://greenkeeper.io/)
+
+[![Standard - JavaScript Style Guide](https://img.shields.io/badge/code%20style-standard-brightgreen.svg)](http://standardjs.com/)
+
+This is a Node.js/Express library for interacting with the [draft store API](https://github.com/hmcts/draft-store).
+The middleware retrieves draft of selected type from the draft store API and sets it in Express.js local user scope.
+If draft does not exist in the draft store then an empty draft is set it in Express.js local user scope.
+
+## Getting started
+
+### How to use it?
+
+#### Installation
+
+To add library to the project dependencies run:
+
+```
+yarn add @hmcts/cmc-draft-store-middleware
+```
+
+#### Sample code
+
+Example use of the middleware is presented below:
+
+```
+import { DraftMiddleware } from '@hmcts/cmc-draft-store-middleware/dist/middleware/draftMiddleware'
```

Comment:
Can we do something to simplify this import statement? Maybe exports are not declared correctly on this module?

Reply:
We could try to get rid of `dist`. Other things has to stay I think.

Reply:
Since this is a project that explicitly delivers the draft middleware I would expect just `import { DraftMiddleware } from '@hmcts/cmc-draft-store-middleware` to work.

I'm pretty sure this should be achievable by providing a proper `index.js` file in module's root.

Reply:
Done